extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -632,7 +662,7 @@ public function canEdit() {
 	 * @return bool
 	 */
 	public function isAuthor( $user ) {
-		if ( $this->mUser ) {
+		if ( $this->loadRevisionsAndAuthor( true ) && $this->mUser ) {
```

Comment:
`$this->loadRevisionsAndAuthor()` does nothing when called for the second time and beyond, if the first call didn't request load from master then the actual data will be coming from slave while we explicitly request for master data here. Worth fixing?